README.md

```diff
@@ -49,6 +49,37 @@ sudo apt-get install g++ qt5-default qttools5-dev-tools qt5-doc qtcreator libglu
 
 You can install [librepcb-git](https://aur.archlinux.org/packages/librepcb-git/) from the AUR.
 
+#### Installation on Mac OS X 10.11
+
+1. Install Xcode through the app store and start it at least once 
+   (for the license)
+2. Install [homebrew](https://github.com/Homebrew/brew) (**the** package 
+   manager)
+3. Install *qt5*
+
+        brew update && brew install qt5
+
+4. Link make the toolchain available (you can undo it by linking against *qt*
+   again)
+
+        brew unlink qt
+        brew link --force qt5
+
+5. Compile everything
+
+        mkdir build && cd build
+        qmake ../librepcb.pro -r
+        make -j8
+        cd ../
+
+6. Run the tests
+
+        ./build/generated/mac/tests
+
+7. Run the program
+
+        open ./build/generated/mac/librepcb.app
```

Comment:
Are the `open` command and the `.app` suffix really required, or would `./build/generated/mac/librepcb` also work?

I think this section "7. Run the program" would be better moved further down in this document, to the already existing section "Run LibrePCB" -> "From command line". If `./build/generated/mac/librepcb` were also working, we could just replace the already existing line:
```
./generated/unix/librepcb
```
with this one:
```
./generated/unix|mac|windows/librepcb
```